<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Initial Panel Widget Allocate Parcels</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="btn_generate_offline_projects">
       <property name="toolTip">
        <string>Generate offline projects</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/Asistente-LADM-COL/resources/images/save.svg</normaloff>:/Asistente-LADM-COL/resources/images/save.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_show_summary">
       <property name="toolTip">
        <string>Show summary</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/Asistente-LADM-COL/resources/images/tasks.png</normaloff>:/Asistente-LADM-COL/resources/images/tasks.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="prb_to_offline">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_reallocate">
       <property name="toolTip">
        <string>Reallocate selected parcels</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/Asistente-LADM-COL/resources/images/deselect.png</normaloff>:/Asistente-LADM-COL/resources/images/deselect.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_allocate">
       <property name="toolTip">
        <string>Allocate selected parcels</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/Asistente-LADM-COL/resources/images/select.png</normaloff>:/Asistente-LADM-COL/resources/images/select.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="chk_show_only_not_allocated">
     <property name="text">
      <string>Show only not allocated parcels</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="content_layout">
     <item row="1" column="0">
      <widget class="QTableWidget" name="tbl_parcels">
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>42</number>
       </attribute>
       <column>
        <property name="text">
         <string>NÃºmero predial</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Estado</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QgsFilterLineEdit" name="mLineEdit">
       <property name="placeholderText">
        <string>Search by parcel code...</string>
       </property>
       <property name="showSearchIcon">
        <bool>true</bool>
       </property>
       <property name="qgisRelation" stdset="0">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_configure_surveyors">
       <property name="toolTip">
        <string>Configure surveyors</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/Asistente-LADM-COL/resources/images/process.svg</normaloff>:/Asistente-LADM-COL/resources/images/process.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
